src/models/felt.rs

```diff
@@ -49,6 +49,14 @@ impl From<Address> for Felt252Wrapper {
     }
 }
 
+#[allow(clippy::fallible_impl_from)]
+impl From<U64> for Felt252Wrapper {
+    fn from(value: U64) -> Self {
+        let felt = FieldElement::from_byte_slice_be(&value.to_be_bytes::<8>()).unwrap(); // safe unwrap since U64 is 8 bytes
```

Comment:
Can't you do something like `U64` to `u64` and then `FieldElement::from(u64)`?

Reply:
Haven't found how to

Reply:
```suggestion
        let felt = FieldElement::from(value.to::<u64>());
```

Reply:
The `to` function can panic because the Uint needs to implement `UintTryTo<T>`, but because there is no risk going from U64 to u64, it won't ever panic.